code/generator/GeneratorVector.cpp

```diff
@@ -60,6 +60,9 @@ bool GeneratorVector::Generate( const genType_t type, const uint32_t numComponen
 		m_codeHeader += "#if defined( __clang__ ) || defined( __GNUC__ )\n";
 		m_codeHeader += "#pragma GCC diagnostic push\n";
 		m_codeHeader += "#pragma GCC diagnostic ignored \"-Wpedantic\"\n";
+		m_codeHeader += "#elif defined( _MSC_VER )\n";
+		m_codeHeader += "#pragma warning( push )\n";
+		m_codeHeader += "#pragma warning( disable : 4201 )\n";
```

Comment:
I'd add a comment on the end of that line just saying what that warning is.  So something like: ```// nameless struct/union```.

Reply:
Added comment in the next commit :)

Reply:
Merged.

I wonder if this also means that Travis VMs on Windows don't have all the warnings turned on.